src/components/Footer/index.jsx

```diff
@@ -15,9 +16,11 @@ class Header extends React.Component { // eslint-disable-line react/prefer-state
             <Link className="logo" to="/">
               <Img src={EdxLogo} alt="edX logo" />
             </Link>
-            <Link className="logo" to="/">
-              <Img src={EnterpriseLogo} alt="Enterprise logo" />
-            </Link>
+            {(this.props.location.pathname.indexOf('/login') === -1) &&
```

Comment:
I'd rather see this conditional based on the `props.logo` passed in rather than making the footer know about the routes. This just feels like too much knowledge for a footer.   Note: the image will come from props when the feature/branding branch lands: https://github.com/edx/edx-portal/compare/feature/branding#diff-60564e5c08a485e77598ff91f67eb3d3R18